app/services/request_password_reset.rb

```diff
@@ -5,7 +5,10 @@
 ) do
   def perform
     if user_should_receive_registration_email?
-      form = RegisterUserEmailForm.new(password_reset_requested: true, analytics: analytics)
+      form = RegisterUserEmailForm.new(
+        password_reset_requested: true, analytics: analytics,
+        attempts_tracker: irs_attempts_api_tracker
```

Comment:
I think this would be a good place for "all on the same line" or "all on different lines"
```suggestion
        password_reset_requested: true,
        analytics: analytics,
        attempts_tracker: irs_attempts_api_tracker,
```

Reply:
Good catch - done!